HIF.md

```diff
@@ -601,41 +601,41 @@ Only present if KEY_IDENTIFY_MODE_INDEX:
 
 ## Packet Filtering
 
-### `0x59 SET_FILTER_DST64`
-
-If `FILTER_BAD_DEST` is set, drop unicast frames which destination is not this
-address. By default, it use the EUI64 returned in `IND_RESET`.
-
- - `uint8_t value[8]`  
+Filter out received packets in the RCP to prevent unecessary
+[`IND_DATA_RX`](#0x13-ind_data_rx) indications.
 
-### `0x5B SET_FILTER_PANID`
+### `0x58 SET_FILTER_PANID`
 
-If `FILTER_BAD_PANID` is set, drop frames which destination is not this PAN ID.
+Refuse frames with an explicit PAN ID different from this value. By default,
+the value is set to `0xffff` which disables the filter.
 
-FIXME: define default value
-
- - `uint16_t value`  
+ - `uint16_t pan_id`  
 
-### `0x5C SET_FILTER_SRC64`
+### `0x5A SET_FILTER_SRC64`
 
-If `FILTER_BAD_SRC64` is set, drop frames coming from denied extended addresses.
+Refuse frames based on the source address. This should only be used in specific
+testing scenarios to force network topologies. By default, the address list is
+empty.
```

Comment:
I believe there are frame with unknown source address in the 15.4 standard (at least they exist in 802.11). I assume they are are not filtered?

Reply:
These frames are currently always refused by the RCP frame parser (they are not used in Wi-SUN). I should probably add some more details as to what is accepted in `REQ_DATA_TX` and `IND_DATA_RX`, similar to the short paragraph about the auxiliary security header.